src/components/Trigger/Trigger.tsx

```diff
@@ -113,14 +113,13 @@ export const Trigger = ({
                         <IconTrashBin size={IconSize.Size12} />
                     </button>
                 )}
-                <button
+                <div
                     {...buttonProps}
```

Comment:
I think we don't even need those buttonProps, do we?

Reply:
Looks like it was added to handle caret positioning but maybe this is outdated now?